rapids-cmake/cpm/spdlog.cmake

```diff
@@ -29,10 +29,23 @@ across all RAPIDS projects.
 
 .. code-block:: cmake
 
-  rapids_cpm_spdlog( [BUILD_EXPORT_SET <export-name>]
+  rapids_cpm_spdlog( [FMT_OPTION <fmt-option-name>]
+                     [BUILD_EXPORT_SET <export-name>]
                      [INSTALL_EXPORT_SET <export-name>]
                      [<CPM_ARGS> ...])
 
+``FMT_OPTION``
```

Comment:
Wouldn't a value of `CXX_FMT` for the `FMT_OPTION` be sufficient for the `std::format` case?

Reply:
Yes, I could add another option I suppose, but calling it `FMT_OPTION` is maybe a bit misleading at that point?

I would also probably call it `STD_FORMAT` instead of `CXX_FMT` just to be a little bit more consistent in mapping the spdlog CMake options to our options here.

Reply:
I was hand waving away the ambgious shortform of `FMT` to be the formatting engine used by spdlog and not to mean the fmt package specifically.

Reply:
Added an option of `STD_FORMAT`